### DISCUSSION TITLE
Cannot initialize Codespace for simple C++ project

### DISCUSSION THREAD

**Discussion by ribtoks** [community/community#3602] [Codespaces] [answered]
Hi

I created a test project for C++ and when attempting to create a codespace, it is stuck in the "Initializing" phase: https://github.com/codespaces/ribtoks-xpiks-workspace-23q4

After 30 seconds or so, it turns into an error `Failed to start Codespace.`. Other projects in my account are able to start a codespace.

I tried (and it does not work):
- deleting all the codespace configuration and starting a bare one
- deleting codespace and starting over (also creating a codespace from github.com/codespaces page)
- renaming the repository
- making sure I have empty codespace slots (e.g. less than 2 active codespaces)

What is wrong with my codespace?

**Comment by PranamBhat** [comment]
Hi @ribtoks 

I have heard this issue long before. Few resolved this by refreshing the browser and clearing browser history/cache.

For more details:

[https://docs.github.com/en/github/developing-online-with-codespaces/configuring-codespaces-for-your-project](https://docs.github.com/en/github/developing-online-with-codespaces/configuring-codespaces-for-your-project)

[https://github.com/features/codespaces](https://github.com/features/codespaces)

**Reply by ribtoks** [reply]
@PranamBhat this has nothing to do with the browser cache or cookies. I contacted support and the issue is with the too big secrets (> 1KB)

**Comment by ribtoks** [comment, marked as answer]
I contacted support and it appeared that issue was with having secrets that were larger than 1KB
